src/misc/documentation_styleguide.md

```diff
@@ -68,7 +68,7 @@ The first level has no indentation, and it’s completely aligned to the left. S
 
 There are three kind of links when cross referencing a document: within a document, within a project group, and outside a project group. The `[displayed text]` should be to the point and within a pair of square brackets. The displayed text must be followed by `(the link)` enclosed in a pair of parenthesis. Follow these formats for each case:
 
-* Reference within the same document: link directly to the anchor, e.g. `[General rules](#general-rules)`. Note that GitHub automatically creates anchors for titles: if the title is *Title Example*, the anchor will be lowercase, with spaces replaced by dashes, i.e. `#title-example`.
+* Reference within the same document: link directly to the anchor, e.g. `[General rules](#general-rules)`. Note that GitHub automatically creates anchors for titles: if the title is *Title Example*, the anchor will be lowercase, with spaces replaced by hashes, i.e. `#title-example`.
```

Comment:
In this one dashes is correct, it refers to the hyphen between title and example.

Reply:
😅 oops, I was distracted by the `#` in the example. Thanks for the review, I just removed this change.